prusa/link/web/main.py

```diff
@@ -546,6 +546,70 @@ def job_info(req):
     return Response(status_code=state.HTTP_NO_CONTENT)
 
 
+@app.route("/api/v1/job/<job_id>", method=state.METHOD_DELETE)
```

Comment:
Use regular expression [filter](http://poorhttp.zeropage.cz/poorwsgi/documentation.html#regular-expression-routes). You can write
``` python
@app.route("/api/v1/job/<job_id:int>", method=state.METHOD_DELETE)
```

Reply:
OK, that's a good point :)